sdmetrics/statistical/bivariate/base.py

```diff
@@ -15,15 +15,15 @@ class BivariateMetric():
     dtypes = []
 
     @staticmethod
-    def metric(real, fake):
+    def metric(real_2d, synthetic_2d):
         """This function is expected to perform a statistical test on the two
         samples, each of which is two-dimensional, and return a tuple containing
```

Comment:
change to "...to perform a statistical test on two columns.."

Reply:
Changed to "This function is expected to perform a statistical test on the two samples, each of which contains two columns, and..."